actions/websocket_actions.test.jsx

```diff
@@ -534,6 +550,12 @@ describe('reconnect', () => {
         reconnect(false);
         expect(syncPostsInChannel).toHaveBeenCalledWith('otherChannel', '12345');
     });
+
+    test('should call loadPluginsIfNecessary when socket reconnects', () => {
+        const {loadPluginsIfNecessary} = require('plugins'); // eslint-disable-line global-require
```

Comment:
I did it like that because I wanted to use the same name (`loadPluginsIfNecessary`) in other tests where this function is not mocked. Do you think I should import it globally and change the other usages to something like `loadPluginsIfNecessaryActual`?

That said, based on this comment I think `Client4` could also be imported globally.

Reply:
Hmm, yeah, that's a problem that comes up a lot with test files like this since they contain a lot of files, some of which want a mock and some of which don't. Maybe it would make sense to split the plugin tests into a separate file from the `reconnect` ones since that would let us mock the function for the `reconnect` tests and not mock it when we're actually testing the plugin-related websocket actions?

Reply:
I guess I could move loadPluginsIfNecessary tests to plugins/tests/. I'm just not sure how to name this file.

Reply:
Something like `websocket_actions_plugins.test.jsx` would work. We've done that elsewhere like with `utils/text_formatting_at_mentions.test.jsx` in the web app